chichiwang/exercise-js
# Serve the `/static` directory statically

## Description
Linked to Issue: #163
Serve static files without extension from the `/static` directory using [serve-static](https://expressjs.com/id/resources/middleware/serve-static.html).

## Changes
* Add dependency `serve-static` to `package.json`
* Replace `express.static` implementation with `serve-static` in `server/index.ts`
* Remove `static/.well_known/health.txt` and replace it with `static/.well_known/acme-challenge/health`

## Steps to QA
* Pull down and switch to this branch
* Run the app and verify in browser that routing to `/.well_known/acme-challenge/health` triggers a download of a file `health` without a file extension
  * File should contain the text "accessible"
